src/components/Container/Container.scss

```diff
@@ -0,0 +1,25 @@
+@import './../../styles/fonts.scss';
+@import './../../styles/breakpoints.scss';
+@import './../../styles/alignment.scss';
+
+.container {
+  display: grid;
+  max-width: $breakpoint-laptop-large;
+  padding: 5px;
+  margin: 0 16px;
+  text-align: center;
+}
+// TODO include an escape before this loop
+$total-columns: 12;
```

Comment:
Lets hoist vars to top of page after imports

Reply:
@mattlaguardia , vars hoisted to top of page after imports.